src/main/java/com/rainbowrave/RainbowRaveConfig.java

```diff
@@ -202,4 +209,15 @@ default boolean recolorLootBeams()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "whichMouseTrailStyle",
+			name = "Mouse Trail Style",
+			description = "Which trail style to use.",
+			position = 16
+	)
+	default MouseTrailStyle whichMouseTrailStyle()
```

Comment:
name this "mouseTrailStyle".

Reply:
Rename the function or the keyName? Any of the other configs that used enums/dropdown were prefaced with which, so I followed that convention.

Reply:
As per Discord conversation, we're going to leave this as is.